airbyte-integrations/connectors/destination-iceberg-v2/src/main/kotlin/io/airbyte/integrations/destination/iceberg/v2/io/IcebergUtil.kt

```diff
@@ -162,27 +179,34 @@ class IcebergUtil {
      * @return The Iceberg [Catalog] configuration properties.
      */
     fun toCatalogProperties(icebergConfiguration: IcebergV2Configuration): Map<String, String> {
+        System.setProperty(
```

Comment:
We should add a comment about why this is necessary for now and what the long term solution is.